### PR TITLE
Fix doc typos

### PR DESCRIPTION
**Pull request by chocolateboy**

### CODE REVIEW

**Comment by dkhamsing** [comment]
@chocolateboy was this automated or manual?

**Comment by chocolateboy** [comment]
Manual.

**Comment by dkhamsing** [comment]
Interesting

While there are some good finds and suggestions, some others I would not change - will leave comments in the `file changed` view

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Verify links in [awesome](status/status.md) projects
 [![Gem Version](https://badge.fury.io/rb/awesome_bot.svg)](https://badge.fury.io/rb/awesome_bot)
 [![](https://img.shields.io/badge/awesome-status-brightgreen.svg)](status/status.md)
 
-`awesome_bot` checks for valid URLs in a file, it can be used to [verify pull requests](#validate-pull-requests) updating a README.
+`awesome_bot` checks for valid URLs in a file. It can be used to [verify pull requests](#validate-pull-requests) which update a README.
```
Comment (dkhamsing) [inline README.md:11]: Not interested in this change, please undo

File: README.md [hunk 2]
```diff
@@ -21,8 +21,8 @@ Verify links in [awesome](status/status.md) projects
 ``` shell
 Usage: awesome_bot [file or files]
        awesome_bot [options]
-    -f, --files [files]              Comma separated files to check
-    -a, --allow [errors]             Status code errors to allow
+    -f, --files [files]              Comma-separated files to check
+    -a, --allow [errors]             Status-code errors to allow
```
Comment (dkhamsing) [inline README.md:25]: Usage comes from the script so please also make the change in the script or undo, thanks

File: README.md [hunk 3]
```diff
@@ -31,17 +31,17 @@ Usage: awesome_bot [file or files]
     -d, --request-delay [seconds]    Set request delay
     -t, --set-timeout [seconds]      Set connection timeout (default: 30)
         --skip-save-results          Skip saving results
-    -w, --white-list [urls]          Comma separated URLs to white list
+    -w, --white-list [urls]          Comma-separated URLs to whitelist
```
Comment (dkhamsing) [inline README.md:34]: Same as above

File: README.md [hunk 4]
```diff
@@ -31,17 +31,17 @@ Usage: awesome_bot [file or files]
...
 ```
 
-- You can check multiple files (comma separated or `*` pattern, look below for details).
+- You can check multiple files (comma-separated or `*` pattern - see below for details).
```
Comment (dkhamsing) [inline README.md:37]: Not interested in the 2nd change after pattern, please undo

File: README.md [hunk 5]
```diff
@@ -69,23 +69,23 @@ Issues :-(
 
 ```shell
 $ awesome_bot README.md --allow-dupe --allow-redirect -w rubydoc,giphy
-# allow redirects, dupes and white list all links matching rubydoc and giphy
+# allow redirects and dupes, and whitelist all links matching rubydoc and giphy
```
Comment (dkhamsing) [inline README.md:72]: Not interested in this change, please undo

File: README.md [hunk 6]
```diff
@@ -230,13 +230,13 @@ i.e.
...
 
-As it happens, the default code snippet provided contain a redirect so adding a badge could fail your status :sob:.. one way to fix this is to white list `travis-ci`, i.e.
+As it happens, the default code snippet provided contains a redirect so adding a badge could fail your status :sob:... one way to fix this is to whitelist `travis-ci`, i.e.
 
 ```
 - awesome_bot README.md --white-list travis-ci
 ```
 
-You can also add a badge for other CI tools, check out [shields.io](https://shields.io/).
+You can also add a badge for other CI tools. Check out [shields.io](https://shields.io/).
```
Comment (dkhamsing) [inline README.md:239]: Not interested in this change, please undo

File: README.md (hunk 1 above)
Comment (chocolateboy) [reply README.md:11]: FYI: https://en.wikipedia.org/wiki/Comma_splice

Comment (dkhamsing) [reply README.md:11]: Thanks, still not interested.

**Pull request closed without merging**